### PR TITLE
Spraytan no longer makes you permanently orange on OD

### PR DESCRIPTION
**Pull request by Mqiib**
# General Documentation

### Intent of your Pull Request

Makes spray tan not irreversibly turn you orange because it's stupid. Other effects are left alone because those can be arguably  funny and aren't PERMANENT

### Why is this change good for the game?

Some people don't want to be orange forever

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Spray tan no longer turns you permanently orange or yellow

# Changelog

he server.

:cl:  

rscdel: Spray tan will no longer turn you irreversibly orange when you overdose on it!

/:cl:

### CODE REVIEW

**Comment by LazennG** [comment]
lizard racism becoming apparent IS funny

**Comment by alexkar598** [comment]
Merge remote-tracking branch 'MqiibUpstream/master'

**Comment by ToasterBiome** [comment]
terrible poopy diaper knee jerk reaction to getting pwned by spraytan

smh u guys

**Comment by boodaliboo** [comment]
bad pr

**Comment by maxion12345** [comment]
get orange'd lmao

**Comment by Jarod1200** [comment]
this is an ided PR i think

**Comment by ChesterTheCheesy** [comment]
"it's my gimmick as clown to make everyone permanently orange as fuck"

**Comment by SomeguyManperson** [comment]
i ded pr :((((

**Comment by Mqiib** [comment]
closed due to #11201
